chia/rpc/wallet_rpc_api.py

```diff
@@ -500,6 +503,62 @@ async def get_transactions(self, request: Dict) -> Dict:
             "wallet_id": wallet_id,
         }
 
+    def __get_csv_line(self, tx, prefix, date_fmt, delim):
+
+        chia_amount = Decimal(int(tx.amount)) / units["chia"]
```

Comment:
What was the rational for using decimal here instead of leaving it as int mojo? We do all accounting with mojos in the source code.

Reply:
Most tax and accounting software that I've seen so far (example: cointracker) uses the main denomination XCH.  My primary use case is to be able to import transactions into software like this readily, without having to do too many transformations.